src/main/scala/viper/gobra/ast/internal/PrettyPrinter.scala

```diff
@@ -233,7 +233,7 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
     case Seqn(stmts) => ssep(stmts map showStmt, line)
     case Label(label) => showProxy(label)
     case If(cond, thn, els) => "if" <> parens(showExpr(cond)) <+> block(showStmt(thn)) <+> "else" <+> block(showStmt(els))
-    case While(cond, invs, body) => "while" <> parens(showExpr(cond)) <> line <>
+    case While(cond, invs, terminationMeasure, body) => "while" <> parens(showExpr(cond)) <> line <>
       hcat(invs  map ("invariant " <> showAss(_) <> line)) <> block(showStmt(body))
 
```

Comment:
the termination measure should be printed with the rest of the loop

Reply:
I will omit comments like this from now on. In general, when the specification of a function, method or loop is printed, its termination measures should also be printed